### PR TITLE
NOISSUE - Fix http request example with metadata query param

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### CODE REVIEW

File: docs/platform-management.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ curl -s -S -i --cacert docker/ssl/certs/ca.crt -H "Authorization: <user_auth_tok
 You can specify `name` and/or `metadata` parameters in order to fetch specific group of things. When specifying metadata you can specify just a part of the metadata JSON you want to match.
 
 ```bash
-curl -s -S -i --cacert docker/ssl/certs/ca.crt -H "Authorization: <user_auth_token>" https://localhost/things?offset=0&limit=5&metadata={"serial":"123456"}
+curl -s -S -i --cacert docker/ssl/certs/ca.crt -H "Authorization: <user_auth_token>" https://localhost/things?offset=0&limit=5&metadata="\{\"serial\":\"123456\"\}"
```
Comment (manuio) [inline docs/platform-management.md:119]: Are you sure that first `\"` is needed?

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/platform-management.md (hunk 1 above)
Comment (mteodor) [reply docs/platform-management.md:119]: Yes, checked it

Comment (manuio) [reply docs/platform-management.md:119]: Why not last one?

**Review by manuio — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**